### ISSUE TITLE
Embeddium crash With Axiom

### ISSUE THREAD

**Issue by Dokman** [FiniteReality/embeddium#311]
### Bug Description

Try to launch Embeddium with axiom

### Reproduction Steps

Update to 3.20 add Sinytra Connector and forgified fabric api and Axiom

### Log File

[crash-2024-06-11_11.42.41-client.txt](https://github.com/user-attachments/files/15792193/crash-2024-06-11_11.42.41-client.txt)
https://mclo.gs/F9fATgb

**Comment by Blackrobe** [comment]
Can confirm, Axiom works with the original Sodium + Indium instead but not the latest Embeddium.

This line can be seen in my own crash report (also in OP's crash report), though not sure if relevant:

```
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Callback method addDrawCommands(Lme/jellysquid/mods/sodium/client/gl/device/MultiDrawBatch;JIIII)Z in axiom.mixins.json:compat.MixinSodiumRegionChunkRenderer from mod axiom failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap axiom-refmap.json
```

https://mclo.gs/MWfByDr

Edit: From Axiom's Discord, there's a thread talking about this titled "Compatibility with Embeddium on 1.20.6". From what I read the Axiom team might be planning for a fix on their side when 1.21 come out.

**Comment by embeddedt** [comment]
https://github.com/embeddedt/embeddium/issues/306#issuecomment-2146392383
